pkg/descheduler/strategies/node_affinity.go

```diff
@@ -26,19 +26,29 @@ import (
 	"k8s.io/api/core/v1"
 )
 
+func Contains(a []string, x string) bool {
```

Comment:
Any specific reason for making this function public?

Reply:
Actually no, changed it.